cmd/gapit/common.go

```diff
@@ -545,6 +550,10 @@ func printCommand(ctx context.Context, client service.Service, p *path.Command,
 				if sl == nil {
 					return fmt.Errorf("Observations are expected to be slices")
 				}
+				if (tm.Range.Size > 1024) {
```

Comment:
I understand the logic to be "if it's bigger than 1k, don't even bother displaying the first 10 values". Yet for the sake of predictability of `gapit commands` output, maybe we want to always display the first 10 values?

Reply:
So agreed, but then we have to actually pull the data back from the server. Which can take a significant amount of time. This was to speed it up, in the cases where we don't even want the data. 

There is a new RPC which will give the size of the underlying object, so we could use that, but it means one more round-trip per request. 

Since this is mostly for testing, I am ok either way if you have a preference.

Reply:
Oh OK I didn't have the data transfer aspect in mind. Let's keep your approach then!